### PR TITLE
Improvements to policy ensemble to fail fast if a Policy class' load function is poorly written (returns None or an object that is not an instance of Policy)

### PR DESCRIPTION
**Pull request by andrewluotechnologies**
Improvements to policy ensemble to fail fast if a Policy class' load function is poorly written (returns None or an object that is not an instance of Policy)

While I was writing my own custom Policy classes I encountered this issue. This isn't actually a bug in Rasa, but rather an enhancement to make the policy loading fail fast - otherwise you only get the error (something like NoneType doesn't have the attribute predict, etc.) later on when you try to pass text to the Rasa Agent. I think it would be useful for debugging to fail fast when the Agent is loaded rather than when it receives its first message.

### CODE REVIEW

File: rasa_core/policies/ensemble.py [hunk 1]
```diff
@@ -180,6 +180,16 @@ def load(cls, path: Text) -> 'PolicyEnsemble':
             dir_name = 'policy_{}_{}'.format(i, policy_cls.__name__)
             policy_path = os.path.join(path, dir_name)
             policy = policy_cls.load(policy_path)
+            if policy is None:
```
[the remark this answers is not in the document]
Comment (andrewluotechnologies) [reply rasa_core/policies/ensemble.py]: Sure, will do.

**Comment by andrewluotechnologies** [comment]
Do you have any particular place that you suggest to add a new test?  I don't see any explicit tests for PolicyEnsemble in tests/test_policies.py but perhaps I'm not looking in the right place.  If not, I can just add a new test for PolicyEnsemble in tests/test_policies.py.

**Comment by tmbo** [comment]
You are right, let's put them into a new file, e.g. `test_ensembles.py`

**Comment by andrewluotechnologies** [comment]
Just checking to see if there's any other comments on this.

**Pull request merged**